### DISCUSSION TITLE
Dual Dimmer not reporing status (HZC Electric D086-ZG)

### DISCUSSION THREAD

**Discussion by rikardronnkvist** [Koenkk/zigbee2mqtt#16200] [Devices]
I installed a HZC Electric D086-ZG dimmer but I can't get it to report back status to HomeAssistant (via MQTT) if I press the actual button and not controlling it via MQTT

Tried to set the Rep Interval set to a low numbers but I only see the last state and brightness that I pushed via MQTT.

Any ideas?

**Comment by franssonjohan** [comment]
Experiencing the same issue with reporting state changes from buttons. 
Any update?

**Reply by rikardronnkvist** [reply]
Sad to say no... :(

**Reply by franssonjohan** [reply]
After my reply on this thread I found this discussion https://github.com/Koenkk/zigbee2mqtt/discussions/13622. Will give it a try later.
